izpack-installer/src/main/java/com/izforge/izpack/installer/gui/InstallerController.java

```diff
@@ -69,8 +73,15 @@ private void run(Runnable action)
             {
                 SwingUtilities.invokeAndWait(action);
             }
+            catch (InvocationTargetException e) {
+                logger.info("InvocationTargetException");
+                Throwable targetException = e.getTargetException();
+                targetException.printStackTrace();
+                throw new IzPackException(targetException);
+            }
             catch (Exception exception)
             {
+                exception.printStackTrace();
```

Comment:
Remove printStackTrace(). If stacktrace is required use Logger.

Reply:
Done